weather/crud.py

```diff
@@ -0,0 +1,108 @@
+from fastapi import HTTPException
+
+from sqlalchemy.orm import Session
```

Comment:
The import of `Session` is not used in the code. Since you're using `AsyncSession` in your CRUD operations, consider removing this import to keep the code clean.